apps/web/src/server/appeals/appeal-details/lpa-questionaire-review/lpa-questionnaire-review.mapper.js

```diff
@@ -0,0 +1,319 @@
+	// TODO: this variable is missing from current data
+	// mappedData.applicantDidEnvironmentStatement = {
+	// 	title: 'Applicant completed Environment Statement',
+	// 	value: convertFromBooleanToYesNo(lpaQuestionnaireData.isEnvironmentalStatementRequired),
+	// 	valueType: 'text',
+	// 	actionText: 'Change',
+	// 	actionLink: '#'
+	// };
+
+	mappedData.hasCompletedAnEnvironmentalStatement = {
+		title: 'Environment Statement Complete',
+		value: convertFromBooleanToYesNo(lpaQuestionnaireData.hasCompletedAnEnvironmentalStatement),
+		valueType: 'text',
+		actionText: 'Change',
+		actionLink: '#'
+	};
+	// TODO: this variable is missing from current data
+	// mappedData.environmentStatementDocument = {
+	// 	title: 'Environment Statement Document',
+	// 	value: lpaQuestionnaireData.documents?.environmentalStatement,
+	// 	valueType: 'link',
+	// 	actionText: 'Details',
+	// 	actionLink: '#'
+	// }
+
+	mappedData.environmentStatementResponsesDocument = {
+		title: 'Environment Statement Responses',
+		value: lpaQuestionnaireData.documents?.environmentalStatementResponses,
+		valueType: 'link',
+		actionText: 'Details',
+		actionLink: '#'
+	};
+
+	mappedData.issuedScreeningOpinion = {
+		title: 'Screening Opinion Issued',
+		value: convertFromBooleanToYesNo(lpaQuestionnaireData.includesScreeningOption),
+		valueType: 'text',
+		actionText: 'Change',
+		actionLink: '#'
+	};
+
+	mappedData.screeningOpinionDocument = {
+		//TODO: determine title for this one
+		title: 'Screening Opinion Document',
+		value: lpaQuestionnaireData.documents?.issuedScreeningOption,
+		valueType: 'link',
+		actionText: 'Details',
+		actionLink: '#'
+	};
+
+	mappedData.screeningDirectionDocument = {
+		title: 'Screening Direction Document',
+		value: lpaQuestionnaireData.documents?.screeningDirection,
+		valueType: 'link',
+		actionText: 'Details',
+		actionLink: '#'
+	};
+
+	mappedData.siteNotice = {
+		title: 'Site Notice',
+		value: lpaQuestionnaireData.documents?.siteNotice,
+		valueType: 'link',
+		actionText: 'Details',
+		actionLink: '#'
+	};
+
+	mappedData.affectsSensitiveArea = {
+		title: 'Affects sensitive area',
+		value: convertFromBooleanToYesNo(lpaQuestionnaireData.isSensitiveArea),
+		valueType: 'text',
+		actionText: 'Change',
+		actionLink: '#'
+	};
+
+	mappedData.sensitiveAreaDetails = {
+		title: 'Sensitive area details',
+		value: lpaQuestionnaireData.sensitiveAreaDetails,
+		valueType: 'text',
+		actionText: 'Change',
+		actionLink: '#'
+	};
+
+	mappedData.meetsOrExceedsThresholdOrCriteriaInColumn2 = {
+		title: 'Meets or exceeds threshold or criteria in column 2',
+		value: convertFromBooleanToYesNo(
+			lpaQuestionnaireData.meetsOrExceedsThresholdOrCriteriaInColumn2
+		),
+		valueType: 'text',
+		actionText: 'Change',
+		actionLink: '#'
+	};
+
+	return mappedData;
+}
+/**
+ *
+ * @param {any} mappedData
+ * @returns
+ */
+function householderConstraintsLpaQuestionnaire(mappedData) {
+	let header = '1. Constraints, designations and other issues';
```

Comment:
can be const, as can sectionData - and the same in the other householder function